### ISSUE TITLE
Nested mixins

### ISSUE THREAD

**Issue by jessehattabaugh** [andyjansson/postcss-sassy-mixins#2]
``` css
@mixin foo {
    color: red;
}

@mixin bar {
    @include foo;
    background-color: blue;
}

.baz {
    @include bar;
}

/* Result */
.baz {
    @include foo;
    background-color: blue;
}

/* Expected Result */
.baz {
    color: red;
    background-color: blue;
}
```

**Comment by dan-gamble** [comment]
Would love to see this too :)

An example i've been using:

``` scss
.class {
  @include Button;
  @include Button-Primary;
}
```

Would be nice to be able to use:

``` scss
@mixin Button-Primary {
  @include Button;
  stuff...
}
```

**Comment by andyjansson** [comment]
Fixed in version 2.0.
